qemux86-64.robot

```diff
@@ -33,6 +33,14 @@ Check host OS fingerprint file
   Verify resin-root.fingerprint in "${path_to_fingerprint}"
   Unmount "${mount_destination}"
   Detach loop device "${path_to_loop}"
+Get host OS version of the image
+  Set up loop device for "${image}"
+  Find the last loop device that was assigned
+  Set Suite Variable    ${path_to_loop}    /dev2/loop${LOOPDEVICE}
+  Mount "${path_to_loop}p${host_os_partition}" on "${mount_destination}"
+  Get the host OS version of the image
```

Comment:
Instead of setting the os_version Suite variable inside the keyword - lets set it here?